src/models/review.ts

```diff
@@ -62,7 +62,9 @@ const Review = db.define('review', {
         allowNull: true
     }
 }, {
-    timestamps: false,
+    createdAt: 'insert_date',
+    updatedAt: 'update_date',
+    timestamps: true,
```

Comment:
You're trying to create "createdAt" and "updatedAt" in the database? Because that's what it's going to do

Reply:
No, whenever a record is inserted or updated, the dates with on the table will be updated